joomla/joomla-cms
# Improve clarification on how to use the banned fields in Contacts.

#### Summary of Changes

When users are editing a contact on the Form tab they can add banned email/subject/words. It is unclear how to separate multiple values. This change updates the tooltip with this information.
#### Testing Instructions
1. Go to Components -> Contacts
2. Click on New
3. Go to the Form tab and check the tooltip for Banned Email, Banned Subject and Banned Text. There is no mentioning on how to enter multiple values.
4. Apply the patch
5. Go to Components -> Contacts
6. Click on New
7. Go to the Form tab and check the tooltip for Banned Email, Banned Subject and Banned Text. The tooltip now shows how to enter multiple values.
